### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by sarah-svg**

### CODE REVIEW

File: test/utils.test.js [hunk 1]
```diff
@@ -41,15 +43,30 @@ test('should take in a cartItem and return a tr element with the apporiate conte
...
-        
-    const expected = 15;
-    const actual = calculateItem(cartItemQuantity, cartItemPrice);
+    const expected = 3;
+    const actual = (cartItemQuantity, cartItemPrice);
 
     expect.equal(actual, expected);
   
 });
+//////////// PASS
```
Comment (breeannb) [inline test/utils.test.js:57]: looks like tests are not yet passing but may be a syntax error

File: cart/cart.js [hunk 2]
```diff
@@ -48,4 +48,5 @@ orderButton.addEventListener('click', () => {
     alert(stringyCart);
     localStorage.clear();
     window.location.href = '/';
```
Comment (breeannb) [inline cart/cart.js:50]: you may have to apply a more relative path for it to work on github.  maybe trying: '../index.html' would work for this one'

File: test/utils.test.js (hunk 1 above)
Comment (sarah-svg) [reply test/utils.test.js:57]: Yes that one no  all other three of four  pass except the last one I keep getting innerhtml issue with that one and it throws off the entire test. But when that test is gone the rest pass not sure what is wrong. Also my pages are not showing up correctly and I am not sure what the problem is.

File: cart/cart.js (hunk 2 above)
Comment (sarah-svg) [reply cart/cart.js:50]: Okay will do it after the presentations I just am confused why it shows up in my live but not on the pages

**Pull request merged**